prez/services/connegp_service.py

```diff
@@ -212,22 +211,51 @@ async def _get_available(self) -> list[dict]:
             }
             for result in repo_response[1][0][1]
         ]
+
         if not available:
+            if self.requested_mediatypes is not None and len(self.requested_mediatypes) > 0:
+                fallback_mediatype = self.requested_mediatypes[0][0]
+            else:
+                fallback_mediatype = None
             if self.listing:
+                # known available mediatypes for the 'mem' profile
+                _available_mem_types = [
+                    "application/ld+json" ,
+                    "application/anot+ld+json" ,
+                    "application/rdf+xml" ,
+                    "text/anot+turtle" ,
+                    "text/turtle"
```

Comment:
The list of available types to match on are pulled from the actual profile definition. https://github.com/RDFLib/prez/blob/cc8c3cac4ab3bc0580e65539f4568c43be28512e/prez/reference_data/profiles/prez_default_profiles.ttl#L114-L119